lib/travis/build/addons/apt_retries.rb

```diff
@@ -0,0 +1,34 @@
+require 'travis/build/addons/base'
+
+module Travis
+  module Build
+    class Addons
+      class AptRetries < Base
+        SUPER_USER_SAFE = true
+
+        def before_configure?
+          config
+        end
+
+        def before_configure
+          sh.echo "Configuring default apt-get retries", ansi: :yellow
+          set_retries = <<-EOF
+          if [[ -d /var/lib/apt/lists && -n $(command -v apt-get) ]]; then
+            cat <<-EOS > 99apt
+Acquire {
+  ForceIPv4 "1";
+  Retries "5";
+  https {
+    Timeout "30";
+  };
+};
+EOS
+            sudo mv 99apt /etc/apt/apt.conf.d
+          fi
+          EOF
+          sh.cmd set_retries
```

Comment:
Why define the string separately and then pass to `sh.cmd`?

Reply:
There is no reason, really.

Reply:
One fewer reading/editing dereference SGTM :skull: